packages/safe-tools-subgraph/src/mapping.ts

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable node/no-unsupported-features/es-syntax */
+import { ScheduledPaymentModuleSetup } from '../generated/ScheduledPaymentModule/ScheduledPaymentModule';
+import { Safe } from '../generated/schema';
+
+export function handleScheduledPaymentModuleSetup(event: ScheduledPaymentModuleSetup): void {
+  let safe = new Safe(event.params.owner.toHex());
+  safe.owner = event.transaction.from;
```

Comment:
the subgraph doesn't have an understanding of the checksum address format, so you might get trolled by this when querying the subgraph. to deal with this I made a util that can perform this for you, this way the addresses in the subgraph are normalized. Here's the util: https://github.com/cardstack/cardstack/blob/main/packages/cardpay-subgraph/src/utils.ts#L228-L245

Reply:
Thanks for describing the issue! I ported over the checksum code.